src/Domain/Model/ApplicantName.php

```diff
@@ -80,6 +86,18 @@ public function setLastName( string $lastName ): void {
 		$this->lastName = $lastName;
 	}
 
+	public function getPersonType(): string {
+		return $this->personType;
+	}
```

Comment:
Do you really need this? It reduces encapsulation by making it clear there is a person type field.